### DISCUSSION TITLE
Who is behind Risor?

### DISCUSSION THREAD

**Discussion by applejag** [deepnoodle-ai/risor#126] [Q&A] [answered]
Is it a company product or hobby project? Are you a group of maintainers, or mainly just one?

Would be nice with a small "about"/"history" section in the docs.
I've seen some mentions of a Tamarin project here and there; what relation does Risor have with that project?

And why was it created? What urge did it fill?

Just curious :)

**Comment by myzie** [comment, marked as answer]
Primarily just me as the maintainer, although it's great to see a growing number of contributors like yourself coming onto the scene 😄

Good suggestion on the about/history section.

I really wanted a project name that matched a domain name for the docs site and also that was a good and short CLI name. I happened upon the name Risor (and the corresponding .io domain) and thought that it would work. Given that, I decided to rename Tamarin to Risor and claim that domain.

The name Tamarin was a nod to the [Monkey Language](https://monkeylang.org/). I never took a languages or compilers course in school but always had some interest in it, so I bought the https://interpreterbook.com/ and worked through that slowly as a hobby. That was how it originated.

My intent is that Risor and its modules will always be open source.

In addition, I may explore having a paid service where Risor-based scripts or workflows could run in the cloud to help with various automation and devops tasks, or help build internal tools.

I love Go and work with it on a daily basis, but it's clunky for use in scripting like situations. I typically resort to Python there, unless it's something simple enough for bash, and have also worked in Python in large applications in the past. So I just had the feeling that a hybrid of the two would have value, especially if it embraced the Go ecosystem fully, and had all the benefits of a single binary distribution. Also, managing dependencies can be a drag and with Risor you can bake a whole bunch of capabilities into the binary. No need to "pip install requests" or similar.
